From 77910a929baaf0ecba893e6e54857315384dc374 Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Thu, 4 Aug 2011 09:59:54 +0800
Subject: [PATCH 42/52] Fix NEED_MULTIPLE_NODES unmet direct dependencies warning

Fix the warning that "(SINGLE_MEMORY_CHUNK) selects NEED_MULTIPLE_NODES
which has unmet direct dependencies (DISCONTIGMEM || NUMA)".

Signed-off-by: Alison Wang <b18965@freescale.com>
---
 mm/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/mm/Kconfig
+++ b/mm/Kconfig
@@ -84,7 +84,7 @@ config FLAT_NODE_MEM_MAP
 #
 config NEED_MULTIPLE_NODES
 	def_bool y
-	depends on DISCONTIGMEM || NUMA
+	depends on DISCONTIGMEM || NUMA || COLDFIRE
 
 config HAVE_MEMORY_PRESENT
 	def_bool y
